### ISSUE TITLE
Source option documentation unclear

### ISSUE THREAD

**Issue by nedbat** [coveragepy/coveragepy#477]
*Originally reported by* **Tom Gurion (Bitbucket: [Nagasaki45](https://bitbucket.org/Nagasaki45), GitHub: [Nagasaki45](https://github.com/Nagasaki45))**

----------------------------------------

I'm new to the project. I tried to understand how to use the `--source` option with multiple packages but couldn't figure it out from [the docs](http://coverage.readthedocs.org/en/coverage-4.1b2/source.html#execution). Stating explicitly that the value should be a **comma seperated** list of directories or package names will make it much clearer.

----------------------------------------
- Bitbucket: https://bitbucket.org/ned/coveragepy/issue/477

**Comment by nedbat** [comment]
*Original comment by* **Ed Johnson (Bitbucket: [edwardotis](https://bitbucket.org/edwardotis), GitHub: [edwardotis](https://github.com/edwardotis))**

----------------------------------------

+1, it's far worse if you have to use the .coveragerc file. For instance, if you want to use the django_coverage_plugin, you have to do use the config file.

If you try to add  an app module like this, it will fail, and will not report a syntax error in the config file.

```
#!python




[run]
source = myapp

It has to be:
[run]
source = 
   myapp
```
a. I think source = module should be supported.
b. syntax errors should be exposed to the user. It's maddening to set this up.
I still can't get my templates to be read for the django_coverage_plugin.

## Please add examples for common configurations.
 ##
https://pypi.python.org/pypi/django_coverage_plugin

**Comment by nedbat** [comment]
@edwardotis I'm not sure what difficulty you are having. It's certainly allowed to use "source = myapp" on one line.  The most common form is "source = ." on one line.  Can you share the exact file that was giving you trouble?

**Comment by nedbat** [comment]
Fixed in [494a5bda95a0 (bb)](https://bitbucket.org/ned/coveragepy/commits/494a5bda95a0)
